docs/develop/security.md

```diff
@@ -21,7 +21,7 @@ npm install -D hardhat-tracer
 and add `hardhat-tracer` to your `config.ts` file,
 
 ```typescript
-import "hardhat-tracer"
+import 'hardhat-tracer';
```

Comment:
I would keep double quotes. We're using those in our examples.

```suggestion
import "hardhat-tracer";
```

Reply:
In the repo config, Prettier is set to enforce single quotes, and it also formats TypeScript code blocks inside Markdown. so it keeps rewriting import to single quotes. I changed the block to text for quick workaround